rero/rero-ils
# statistics: implement basic statistics for librarians

- Adds basic statistics for librarians and system librarians
- Adds view 'Statistics librarian' in Tools

Co-Authored-by: Valeria Granata <valeria@chaw.com>

## Why are you opening this PR?
US Basic statistics
https://tree.taiga.io/project/rero21-reroils/us/1744?milestone=306074

## How to test?

- Log in as admin, librarian or system librarian.
- In section Tools: 
     if logged in as admin there are two options: Statistics billing, Statistics, 
     if logged in as librarian or system librarian there is only Statistics
-  In Statistics there should be the list of statistics from Jan 2021 to Dec 2021
- The monthly statistics contains the statistics for each library: 
    A librarian can see only the statistics for his/her libraries.
    A system librarian can see the statistics for his/her libraries and the libraries of his/her organization. 
    An admin can see the statistics for all libraries. 
- The 'Live values' button calculate the statistics of the current month up to the current day.
- It is possible to download the statistics in .csv format by clicking on the download buttons.
- Permission required should display if attempting to access /stats/ or stats/live when logged in as librarian or system librarian.

## Code review check list
